src/unit/format.js

```diff
@@ -0,0 +1,61 @@
+define([
+	"./get",
+	"../common/format-message"
+], function( unitGet, formatMessage ) {
+
+/**
+ * format( value, unit, options, cldr, globalize )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * FIXME
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ *
+ * OBS:
+ *
+ * Unit Sequences are not implemented.
+ * http://www.unicode.org/reports/tr35/tr35-35/tr35-general.html#Unit_Sequences
+ *
+ * Duration Unit (for composed time unit durations) is not implemented.
+ * http://www.unicode.org/reports/tr35/tr35-35/tr35-general.html#durationUnit
+ */
+return function( value, unit, options, pluralGenerator, cldr, globalize ) {
```

Comment:
Just noticed we're already passing `pluralGenerator` above. But, below we're using `globalize` instance. Let's get the below fixed (and make the calling function passes the correct arguments).